components/src/atoms/message/Message.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../../assets/styles/_globals.scss';
+
+$message__width                     : 320px !default;
+$message__padding                   : 16px 25px 0 25px !default;
+$message__padding-bottom            : 10px !default;
+$message__margin-base               : $spacer--medium 0 0 0 !default;
+$message__font-size                 : $font-size-small !default;
+$message__transition                : $transition-base !default;
+$message__color                     : $white !default;
+$message__color-success             : $black !default;
+$message__background                : rgba(25, 118, 210, 1) !default;
+$message__btn-background            : rgba(255, 255, 255, 0) !default;
+$message__background--success       : rgba(105, 220, 161, 1) !default;
+$message__background--error         : rgba(211, 47, 47, 1) !default;
+$message__border-success            : 1px solid rgba(57, 50, 67, 0.4) !default;
+$message__border-bottom             : 1px solid rgba(255, 255, 255, 0.4) !default;
+$message__box-shadow                : 0 4px 6px 0 rgba(57, 50, 67, 0.3) !default;
+$message__actions-padding           : 18px 0 !default;
+$message__information-font-size     : 15px !default;
+
+
+.message {
+  width: $message__width;
+  height: auto;
+  margin: $message__margin-base;
+  padding: $message__padding;
+  color: $message__color;
+  background-color: $message__background;
+  box-shadow: $message__box-shadow;
+  font-size: $message__font-size;
+  transition: $message__transition;
+
+  &__information {
+    padding-bottom: $message__padding-bottom;
+    border-bottom: $message__border-bottom;
+    font-size: $message__information-font-size;
+    font-weight: $font-weight-bold;
+  }
+
+  &__actions {
+    display: flex;
+    padding: $message__actions-padding;
+
+    &-btn {
+      padding: $reset;
+      color: white;
+      background: $message__btn-background;
+      border: none;
+      font-size: $font-size-base;
+      text-align: center;
+
+      &:hover {
+        cursor: pointer;
+      }
```

Comment:
hmm do we need this one?

Reply:
After removed default-class we dont need it ;)